v22.2/show-system-grants.md

```diff
@@ -0,0 +1,96 @@
+---
+title: SHOW SYSTEM GRANTS
+summary: The SHOW SYSTEM GRANTS statement lists the system privileges granted to users.
+keywords: reflection
+toc: true
+docs_area: reference.sql
+---
+
+{% include_cached new-in.html version="v22.2" %} The `SHOW SYSTEM GRANTS` [statement](sql-statements.html) lists the [system privileges](security-reference/authorization.html#supported-privileges) [granted](grant.html) to [users](security-reference/authorization.html#sql-users).
+
+## Syntax
+
+Use the following syntax to show the [system privileges](security-reference/authorization.html#supported-privileges) granted to users:
+
+~~~
+SHOW SYSTEM GRANTS [FOR <users...>]
+~~~
+
+## Parameters
+
+Parameter    | Description
+-------------|-----------------------------------------------------------------------------------------------------
+`users`      | The [user](security-reference/authorization.html#sql-users), or comma-separated list of users, whose system privileges you want to show.
+
+## Response
+
+### Privilege grants
+
+The `SHOW SYSTEM GRANTS` statement returns the following fields:
+
+Field            | Description
+-----------------|-----------------------------------------------------------------------------------------------------
+`grantee`  | The name of the user.
+`privilege`  | The name of the [system privilege](security-reference/authorization.html#supported-privileges) granted to the user.
+`is_grantable`   | `t` (true) if the user has the grant option on the object; `f` (false) if not.
+
+## Required privileges
+
+- No [privileges](security-reference/authorization.html#supported-privileges) are required to use `SHOW SYSTEM GRANTS`.
+
+## Examples
+
+### Show all system grants
+
+To list all system grants for all users and roles:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> SHOW SYSTEM GRANTS;
```

Comment:
Suggestion: (not a blocker) it could be helpful listing out a few more system privileges in the example below, especially to illustrate new ones that don't correspond to an existing role option, e.g. 
- VIEWDEBUG
- VIEWCLUSTERMETADATA

Reply:
Great idea! Done.